.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "rust-analyzer.linkedProjects": [
-    "./app/gui2/rust-ffi/Cargo.toml"
```

Comment:
These fixes in `settings.json` were needed to be able to use the Rust extensions of VS Code. Without changes it was crashing upon project import.